src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -501,6 +512,15 @@ private void updateButtons() {
             downloadButton.setVisible(true);
             cancelDownloadButton.setVisible(false);
         }
+
+        // if less than 200MB available
+        if (downloadDirectory.getUsableSpace() <= 200 * 1000000) {
+            warningButton.setVisible(true);
+            warningButton.setTooltip(new Tooltip(BundleUtils.getLabel("message_warning_lowOnSpace")));
+            logger.warn(BundleUtils.getLabel("message_warning_lowOnSpace"));
```

Comment:
The logged warning should contain some more info on how much space is actually left on disk, or how much free space is required to successfully perform the action.